Header header
uint8 address
uint8 remote_address
float32[] state
float32[] covariance
